### ISSUE TITLE
Update.php error on nginx not addressed in core successfully

### ISSUE THREAD

**Issue by ttrowell** [pantheon-systems/drops-8#97]
My original bug was apparently a duplicate:
https://www.drupal.org/node/2590345

"Expecting:
A message that thee are no pending updates if none are available. Otherwise, the updates are made.

Actual:
After enabling core modules there is a Page Not Found error when running update.php.
The url is /update.php/selection

Error in action:
https://youtu.be/Hbc7Nv6nf-k

It may be that there are no pending updates - but I am not sure."

This issue has not yet been resolved:
https://www.drupal.org/node/2353907#comment-10437205

I do not know if we have made any modifications to nginx as referenced here in comment 23:
https://www.drupal.org/node/2353907#comment-10431773

**Comment by ttrowell** [comment]
I logged this because not being able to run update.php is a big problem and I am not sure where the fix will come from - us or them.  :)

**Comment by joshkoenig** [comment]
Possible fix via nginx config:

https://www.drupal.org/node/2353907#comment-10431773

**Comment by greg-1-anderson** [comment]
Just tried to run update.php on Pantheon; looks like the fix suggested in the link that @joshkoenig referenced will work fine.

**Comment by greg-1-anderson** [comment]
Fix submitted: https://github.com/pantheon-cookbooks/endpoint/pull/98

**Comment by jfrederick** [comment]
In preparation to move to the newly-released D8 RC1 environment, I am working to upgrade my beta15 installation to rc1 in the old Drupal-testing environment. So, I did this:

1. Upgrade code from beta15 to RC1, push code to Pantheon.
2. Now I want to run the database updates. Since Terminus won't work on the Drupal-testing installation anymore, giving the message "Drush 7.0.0 does not support Drupal 8." (https://github.com/pantheon-systems/drops-8/issues/95), I will try to use update.php. But when I hit update.php, I run into the issue in this ticket. Specifically I cannot access update.php/selection, as described in #27 here https://www.drupal.org/node/2353907#comment-10437205 : "After upgrading to the latest RC1 release I went straight to the update.php page. Got the 404 that is trying to access /update.php/selection"

Will the resolution of this ticket fix this problem? Thanks!

**Comment by greg-1-anderson** [comment]
Yes; however, the fix I submitted does not work. We are still working on a solution.

**Comment by greg-1-anderson** [comment]
@jfrederick: as a workaround, you may now use `drush @pantheon.sitename.dev updatedb` to update your database. Also, if you update your Drupal code to 8.0.0-rc1 via the dashboard, instead of doing it locally and pushing the code, then you will also have the opportunity to run updatedb at that time.

**Comment by ttrowell** [comment]
Just documenting that this is still an issue when running update.php after updating to rc3.
http://screencast.com/t/UUFuw2HSoAa

Workaround still stands: use ```drush @pantheon.sitename.dev updatedb```
